FirebaseDatabase/Tests/Unit/FUtilitiesTest.m

```diff
@@ -115,7 +115,7 @@ - (void)testUserAgentString {
   XCTAssertEqualObjects(parts[0], @"Firebase");
   XCTAssertEqualObjects(parts[1], kWebsocketProtocolVersion);   // Wire protocol version
   XCTAssertEqualObjects(parts[2], [FIRDatabase buildVersion]);  // Build version
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IPHONE && !TARGET_OS_WATCH
```

Comment:
I'm curious if the tests can be run on watchOS with appropriate Xcode version?

Reply:
Yes they can, I tested it locally and they all passed (this change was required to make them pass). Once we have it available on GHA I can add it!

Reply:
Great!